modules/mobile/config/routes.rb

```diff
@@ -49,7 +49,27 @@
     get '/letters/beneficiary', to: 'letters#beneficiary'
     post '/letters/:type/download', to: 'letters#download'
     get '/maintenance_windows', to: 'maintenance_windows#index'
+    get '/messaging/health/folders', to: 'folders#index'
+    post '/messaging/health/folders', to: 'folders#create'
+    get '/messaging/health/folders/:folder_id/messages', to: 'messages#index'
+    get '/messaging/health/folders/:folder_id/threads', to: 'threads#index'
+    get '/messaging/health/folders/:id', to: 'folders#show', as: 'folder'
```

Comment:
I decided to not name each route that previously had a name. Only the ones that need a name to fetch the url for a route in some serializers.